src/Maker/MakeRegistrationForm.php

```diff
@@ -77,6 +80,11 @@ public function interact(InputInterface $input, ConsoleStyle $io, Command $comma
             ->addArgument('user-class')
             ->addArgument('username-field')
             ->addArgument('password-field')
+            ->addArgument('will-verify-email', null, '', false)
```

Comment:
Are the extra args needed?

Reply:
short answer - without `false` calling `$input->getArgument('will-verify-email')` will return `null` if the arg is not set.  things break when I remove the extra args.

Long answer - i'll go through and see if we can change something up without adding `$input->setArg..('will-verify-email', false)` right off the bat

Reply:
done - i'm predicting value objects in the future